index.html

```diff
@@ -2,45 +2,59 @@
 <html>
   <head>
     <title>Harmony | Sign Up</title>
+    <link rel="stylesheet" href="./index.css" />
   </head>
   <body>
-    <form class="form-wrapper">
+    <form class="form-wrapper" id="input-form">
       <div class="form-section">
+        <img
+          class="form-top-image"
+          src="./assets/harmony-music-academy-logo.svg"
+          alt="Harmony Music Academy logo"
+        />
+
         <div class="form-field">
           <label for="email" class="form-label">Email*</label>
           <input
             name="email"
             type="text"
             class="form-input"
             placeholder="Your email"
+            id="emailInput"
           />
         </div>
+
         <div class="form-field">
           <label for="level" class="form-label">Level*</label>
-          <select name="level" class="form-input">
+          <select name="level" class="formInput" id="level-input">
```

Comment:
- Change this back to `class="form-input"` to ensure the select box is styled correctly (this change has broken the style as the application can't find a corresponding CSS class called `formInput`)
- Your `id` attribute should be set to simply `level` here